parser_actions.go

```diff
@@ -95,8 +95,10 @@ func (ap *AnsiParser) csiDispatch() error {
 }
 
 func (ap *AnsiParser) print() error {
-	logger.Infof("AnsiParser::print %#x", ap.context.currentChar)
```

Comment:
The logger statements are responsible for a significant amount of the CPU time, even when logging is disabled. We either need to removing the logging or change the logging configuration so that it doesn't burn so much CPU.